cluster/gce/addons/konnectivity-agent/konnectivity-agent-ds.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
       containers:
-        - image: registry.k8s.io/kas-network-proxy/proxy-agent:v0.31.2
+        - image: registry.k8s.io/kas-network-proxy/proxy-agent:v0.34.0
```

Comment:
https://github.com/kubernetes/k8s.io/pull/8719 needs to merge before this can work